src/components/GameView/GameUnavailableOverlay.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <v-overlay
+    data-cy="unavailable-game-overlay"
+    id="unavailable-game-scrim"
+    v-model="unavailableGame"
```

Comment:
Instead of a 2-way v-model binding let's just use a one-way model-value binding:
```suggestion
    :model-value="show"
```

Reply:
v-model is for when you want to set the child's model value from the parent but also sync up the parent's variable with the child when the child emits an update, which we don't need to do here. This is more straightforward top-down prop assignment